### DISCUSSION TITLE
Nested domain capability in srw app

### DISCUSSION THREAD

**Discussion by XiaSun-Atmos** [ufs-community/ufs-srweather-app#651] [Q&A] [answered]
Hi,
Does the srw app support nested domain configuration?
Thanks,
Xia

**Comment by ShawnCebulaNOAA** [comment]
Hello Xia!

Thanks for the question, I'm not sure if it does support nested domains so I've sent the question on to our team so they can answer for you.

Shawn Cebula, EPIC End User Support

**Comment by XiaSun-Atmos** [comment]
Thank you, Shawn! Looking forward to more information.

**Comment by gspetro-NOAA** [comment, marked as answer]
@XiaSun-Atmos The SRW App does not yet support nested domains. I believe the Hurricane App may have (or be developing) that capability, so you may want to check out the [hafs-community](https://github.com/hafs-community/HAFS) repo. EPIC is not yet supporting HAFS, but that will hopefully be coming soon, and in the meantime, the developers there may be able to assist you. 

Best,
Gillian

**Reply by XiaSun-Atmos** [reply]
Thanks for the information @gspetro-NOAA
